backend/src/services/super-admin/super-admin-service.ts

```diff
@@ -374,6 +415,14 @@ export const superAdminServiceFactory = ({
       gitHubAppConnectionSettingsUpdated = true;
     }
 
+    let envOverridesUpdated = false;
+    if (data.envOverrides !== undefined) {
+      const encryptedEnvOverrides = encryptWithRoot(Buffer.from(JSON.stringify(data.envOverrides)));
+      updatedData.encryptedEnvOverrides = encryptedEnvOverrides;
+      updatedData.envOverrides = undefined;
+      envOverridesUpdated = true;
+    }
+
```

Comment:
thoughts on adding a validation step here by checking against the envSchema before saving? This ensures that sync works for the most part and not just fail silently.

<img width="1059" alt="image" src="https://github.com/user-attachments/assets/a0fb0791-eea2-43c0-ae81-5755add0642c" />